app/controllers/shared/api.php

```diff
@@ -318,6 +318,39 @@
         }
     });
 
+App::shutdown()
+    ->groups(['session'])
+    ->inject('utopia')
+    ->inject('request')
+    ->inject('response')
+    ->inject('project')
+    ->inject('dbForProject')
+    ->action(function (App $utopia, Request $request, Response $response, Document $project, Database $dbForProject) {
+        $sessionLimit = $project->getAttribute('auths', [])['maxSessions'] ?? APP_LIMIT_USER_SESSIONS;
+        $session = $response->getPayload();
```

Comment:
Does this works well for OAuth sessions as well?

Reply:
it should as I added the group in the endpoint that actually creates the session object